docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,258 @@
+========
+Artifact
+========
+
+A data artifact is a bundle of input data associated with a particular public health model. It is typically
+stored as an ``hdf`` file on disk with very particular formatting. This file is then used by the 
+vivarium_public_health simulations to fill in all the relevant parameter data. 
+
+It is frequently useful to be able to view or modify this data outside the simulation.  The 
+vivarium_public_health :class:`vivarium_public_health.dataset_manager.artifact.Artifact` provides
+a high level interface to do just that. In this tutorial we'll go through how to view, delete, 
+and write data to an artifact using the tools provided by the
+:class:`vivarium_public_health.dataset_manager.artifact.Artifact`. You'll access data in the artifact through keys,
+mirroring the underlying hdf storage of artifacts.
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+Creating an artifact
+---------------------
+To view an existing hdf file via the :class:`vivarium_public_health.dataset_manager.artifact.Artifact` tools, we'll
+create a new artifact. We can print the resulting artifact to view the tree structure of the keys
+in our artifact. We'll use our test artifact to illustrate:
+
+.. code-block:: python
+
+    from vivarium_public_health.dataset_manager import Artifact
+
+    art = Artifact('test_artifact.hdf')
+    print(art)
+
+::
+
+    Artifact containing the following keys:
+    metadata
+            keyspace
+            locations
+            versions
+    population
+            age_bins
+            structure
+            theoretical_minimum_risk_life_expectancy
+
+Now we have an :class:`vivarium_public_health.dataset_manager.artifact.Artifact` object, which we can use to interact
+with the data stored in the hdf file with which we created it.
+
+Filter Terms
++++++++++++++
+The data stored in artifacts may be large, potentially on the order of millions of rows for a single dataset, and
+loading a full dataset requires time and memory, both of which may be limited. If you are only interested in certain
+subsets of the data, e.g., only years after 2000 or only draw 0, you may want to read only the portion you
+need. This is the idea behind filter terms. Filter terms are built into an Artifact on its creation and apply to all
+data loaded from that Artifact. You can think of filter terms as somewhat similar to the :func:`pandas.DataFrame.query`
+method, although the key difference is that filter terms apply to what data is actually read off disk. This
+ means that they can reduce the time and memory required to load a single dataset from an Artifact. For example, if you
```

Comment:
there is an extra space here, FYI :grin:

Reply:
oh good catch. sphinx does weird formatting if a line starts with a space. just removed it